src/test/unit/StrategyManagerUnit.t.sol

```diff
@@ -146,9 +104,9 @@ contract StrategyManagerUnitTests is Test, Utils {
             )
         );
         dummyToken = new ERC20Mock();
-        dummyStrat = deployNewStrategy(dummyToken, strategyManager, pauserRegistry, dummyAdmin);
-        dummyStrat2 = deployNewStrategy(dummyToken, strategyManager, pauserRegistry, dummyAdmin);
-        dummyStrat3 = deployNewStrategy(dummyToken, strategyManager, pauserRegistry, dummyAdmin);
+        dummyStrat = _deployNewStrategy(dummyToken, strategyManager, pauserRegistry, dummyAdmin);
+        dummyStrat2 = _deployNewStrategy(dummyToken, strategyManager, pauserRegistry, dummyAdmin);
+        dummyStrat3 = _deployNewStrategy(dummyToken, strategyManager, pauserRegistry, dummyAdmin);
 
         // whitelist the strategy for deposit
         cheats.startPrank(strategyManager.owner());
```

Comment:
Nit: pluralize `_strategy` array. Also, we can use just cheats.prank since there's only one call that needs to be pranked

Reply:
pluralized